src/parser/regex/execRegex.ts

```diff
@@ -7,9 +7,12 @@ function execRegex(value: string, regex): string {
     if (REGEXES[regex]) {
       $regex = REGEXES[regex];
     }
-  } else {
+  } else if(regex.pattern) {
     const { pattern, flags } = regex;
     $regex = new RegExp(pattern, flags);
+
+  }else{
```

Comment:
Maybe checking whether the incoming regex object contains a method named exec can prevent some problems that may occur.

```js
} else if (regex?.exec) {
```